tests/test_helpers/test_helpers.ml

```diff
@@ -34,6 +35,12 @@ and pp_module_diff fmt { mname; mdiff } =
       Format.fprintf fmt "Module %s: {Modified (%a)}" mname
         pp_module_modification mdiff
 
+and pp_class_diff fmt { cname; cdiff } =
+  match cdiff with
+  | Added _ -> Format.fprintf fmt "Class (%s, Added)" cname
+  | Removed _ -> Format.fprintf fmt "Class (%s, Removed)" cname
+  | Modified _ -> ()
```

Comment:
Even though this case is never encountered, at least print something!

Reply:
Sure,  i would work on writing some tests. I'm currently working on making the requested modification.